packages/medical-domain/src/domains/repositories/medical/medical-data.service.ts

```diff
@@ -355,11 +266,132 @@ class MedicalDataService {
         } else {
           message = String(error)
         }
-        console.debug({ message, rawData: raw })
+        console.log({ message, rawData: raw })
```

Comment:
We usually avoid having "console.log" (it often pollute the console with information that does not feel important). In this case, I think that it is better to use "console.error'
```suggestion
        console.error({ message, rawData: raw })
```